test.sh

```diff
@@ -71,6 +61,8 @@ node build-nightly.js
 
 echo
 echo 'saving the results...'
+git config user.email "hubbed@kap.co"
+git config user.name "Imma Bot"
```

Comment:
How do permissions work on actions? On Heroku I had to set an env variable `$GITHUB_TOKEN`